src/main/python/linuxband/gui/events/groove.py

```diff
@@ -39,17 +39,17 @@ def __init__(self, glade, grooves):
 
     def on_treeview1_cursor_changed_callback(self, treeview):
         """ User clicked on the groove in the first column. """
-        path, column = treeview.get_cursor() #@UnusedVariable
+        path = treeview.get_cursor()[0]
         gr = self.__groovesModel[path[0]]
```

Comment:
This change looks suspicious. Line 43 is still using index on the path variable.

Reply:
According to the [Treeview documentation](https://developer.gnome.org/pygtk/stable/class-gtktreeview.html#method-gtktreeview--get-cursor) get_cursor() returns a tuple as (path, column) where path is a TreePath object, which itself is always a tuple as stated in the [TreeModel Interface docs](https://docs.huihoo.com/pygtk/2.0-tutorial/sec-TreeModelInterface.html). So accessing path[0] seems perfectly valid to me.

Reply:
Thanks for the info. Works great.